tests/h/views/main_test.py

```diff
@@ -12,36 +12,34 @@
 from h.views import main
 
 
-@mock.patch('h.client.render_app_html')
 @pytest.mark.usefixtures('routes')
-def test_og_document(render_app_html, annotation_document, document_title, pyramid_request, group_service, links_service):
+def test_og_document(annotation_document, document_title, pyramid_request, group_service, links_service):
     annotation = Annotation(id='123', userid='foo', target_uri='http://example.com')
     context = AnnotationResource(annotation, group_service, links_service)
     document = Document()
     annotation_document.return_value = document
     document_title.return_value = 'WikiHow — How to Make a ☆Starmap☆'
 
-    render_app_html.return_value = '<html></html>'
-    main.annotation_page(context, pyramid_request)
-    args, kwargs = render_app_html.call_args
-    test = lambda d: 'foo' in d['content'] and 'Starmap' in d['content']
-    assert any(test(d) for d in kwargs['extra']['meta_attrs'])
+    ctx = main.annotation_page(context, pyramid_request)
+
+    def test(d):
```

Comment:
The purpose of this test is to make sure that the right `<link>` and `<meta>` tags are rendered when you visit `/a/<annotation ID>` pages, depending on the data available in the annotation. Probably easier to clarify by changing the test name as much as this lambda. I'm going to leave it for the moment to get the rest of the work here merged.